src/buttons/CopyToClipboardButton/Readme.md

```diff
@@ -0,0 +1,29 @@
+# CopyToClipboardButton
+
+Button that copy the given value to the clipboard and provide 3 steps feedback for user
```

Comment:
(sp) should be: `A button that copies...` and `... and provides 3 steps of feedback to the user.`

Reply:
🇺🇸 🇬🇧 thanks!